### DISCUSSION TITLE
Why do I loose the colour of my subtitles (srt) ?

### DISCUSSION THREAD

**Discussion by Frau-Wunderlich** [mifi/lossless-cut#1792] [Q&A] [answered]
Hallo :)
I use LC (Great Software btw)  mostly to add subtitles to mp4 Videos with Audio. 
I dont want to burn the text in, just ,well I guess softcode is the right term.
(Before I used YAMB , but it's getting old.)

Works fine, but the colour-information gets lost somehow.
In the srt it looks like this:

------------------------------------------
```
1
00:00:04,600 --> 00:00:07,440
<font color="#00ff00ff">Hey, you! Stop!</font>

2
00:00:16,760 --> 00:00:19,320
<font color="#ffffffff">Never!</font>

```
------------------------------------------
The finished video shows just the standard white font.
I really tried, but I am just a user....I can not find the answer.
Please can someone help?

Is there a way, or can't it be done with LosslessCut?

....and sorry for the clumsy english.

**Comment by mifi** [comment]
Hi. Do you know for sure that mp4 supports colored subtitles? Do you have an example such file with colored subtitles?

**Reply by Frau-Wunderlich** [reply, marked as answer]
Hi mifi, thanks for your answer.

Oh , verdorrich.
I tested the last 3 hours, and no.. the coloured subtitles just work in the Player, when the srt is an own file.
I remembered wrongly.
Sorry for wasting your time.

I will use LC now, since Yamp has problems with newer codecs(?).
Still have to get used to the GUI, but will work my way through it.
LC saved me with a livestream-recording what was far to big for all my tools to even open it.
Your program not even stuttert. It.s great. 
Thank you.

**Reply by mifi** [reply]
Ok no worries, thanks for reporting back
